arangod/Statistics/StatisticsWorker.cpp

```diff
@@ -923,6 +923,10 @@ void StatisticsWorker::generateRawStatistics(VPackBuilder& builder, double const
   // _serverStatistics()
   builder.add("server", VPackValue(VPackValueType::Object));
   builder.add("uptime", VPackValue(serverInfo._uptime));
+  builder.add("transactionsStarted", VPackValue(serverInfo._transactionsStatistics._transactionsStarted));
```

Comment:
I would prefer if we used an object with sub-attributes "started", "aborted", "committed", "intermediateCommits" here.
That would be more in line with, for example, the scheduler and the v8 context statistics which also use objects with sub-attributes.